Makefile

```
@@ -1189,3 +1192,15 @@ docs/serve:
 		-v ${PWD}:/docs \
 		--entrypoint sh $(MKDOCS_IMAGE) \
 		-c 'mkdocs serve -s --dev-addr=0.0.0.0:$(MKDOCS_SERVE_PORT) -f mkdocs.yml'
+
+# k3d/generate-user-keys will generate seed user ssh keys that can be used for local testing
+SEED_USERS = guest reporter developer maintainer owner orgviewer orgadmin orgowner platformorgowner platformviewer platformowner
```

I believe this is missing an `orguser` user:
```suggestion
SEED_USERS = guest reporter developer maintainer owner orguser orgviewer orgadmin orgowner platformorgowner platformviewer platformowner
```